config.js

```diff
@@ -128,13 +128,8 @@ module.exports = (function () {
     PIX_APPS_ENVIRONMENTS: ['integration', 'recette', 'production'],
     PIX_TUTOS_REPO_NAME: 'pix-tutos',
     PIX_TUTOS_APP_NAME: 'pix-tutos',
-    PIX_GRAVITEE_APIM_REPO_NAME: 'pix-gravitee-apim',
-    PIX_GRAVITEE_APIM_APPS_NAME: [
-      'pix-gravitee-apim-portal-ui-production',
-      'pix-gravitee-apim-gateway-production',
-      'pix-gravitee-apim-console-ui-production',
-      'pix-gravitee-apim-rest-api-production',
-    ],
+    PIX_APIM_REPO_NAME: 'pix-nginx-apim',
+    PIX_APIM_APPS_NAME: ['pix-nginx-apim-production'],
```

Comment:
```suggestion
    PIX_APIM_APP_NAME: 'pix-nginx-apim-production',
```
Comme nous n'avons plus qu'une application, on pourrait repasser tout au singulier, et enlever le tableau comme sur airflow par exemple